shared/src/bundles/user-details/types/user-details-approve-request-dto.type.ts

```diff
@@ -0,0 +1,7 @@
+type UserDetailsApproveRequestDto = {
+    id: string;
+    isApproved: boolean;
+    deniedReason?: string;
+};
+
+export { type UserDetailsApproveRequestDto };
```

Comment:
You don't need this in this PR. This is another task. Please don't go beyond the scope of your task !